docs/pages/cdp/cdp-Installing.md

```diff
@@ -24,40 +24,28 @@ toc: false
     sudo npm install -g @brightside/core
     ```
 
-3. Build and install the cics-deploy plug-in as described in [Setting up your development environment](https://github.com/IBM/zowe-cli-cics-deploy-plugin/blob/master/docs-internal/tutorials/Setup.md). For example:
+3. Install the cics plugin:
 
     ```console
-    mkdir ~/cics-deploy
-    cd ~/cics-deploy
-    rm -Rf *
-    git clone https://github.com/IBM/zowe-cli-cics-deploy-plugin
-    cd zowe-cli-cics-deploy-plugin
-    npm install
-    npm run build
-    zowe plugins install .
+    zowe plugins install @brightside/cics
     ```
 
-4. Install the cics plugin:
+4. Install the cics-deploy plugin:
 
     ```console
-    zowe plugins install @brightside/cics
+    zowe plugins install zowe-cli-cics-deploy-plugin
     ```
 
-5. Verify the plug-ins are installed:
+5. Validate the plug-ins are installed:
 
     ```console
-    zowe plugins list
+    zowe plugins validate
```

Comment:
I suggest reverting this to `zowe plugins list` as being more useful as it summarises the version and registries for each plugin. The validate is automatically run as part of the previous `zowe plugins install ...`.

Reply:
done. 
also changed the output to relevant one.